Sources/Bridges/OTelSwiftLog/LogHandler.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import Logging
+
+// let the bridgename be the url of the package?
+let bridgename: String = "OTelSwiftLog"
+let version: String = "1.0.0"
+
+public struct InstrumentationScope{
+    public var name: String
+    public var version: String?
+    public var eventDomain: String?
+    public var schemaUrl: String?
+    public var includeTraceContext: Bool?
+    public var attributes: [String: AttributeValue]?
+}
+
+// Define a custom log handler
+struct OTelLogHandler: LogHandler {
+    private var scope: InstrumentationScope // Property to store instrumentation scope name
+    private var loggerProvider : LoggerProvider  // Property to set LoggerProvider
+    private var logger: OpenTelemetryApi.Logger 
+
+    // Define the log level for this handler
+    private var _logLevel: Logging.Logger.Level = .info
```

Comment:
Why a private and a public that overwrites it?

Reply:
It is so that the user can change the default log level.

Reply:
Then make it directly public